Build failure: python3Packages.occ-core

### Steps To Reproduce

Steps to reproduce the behavior:
1. `nix-build '<nixpkgs>' -A pythonocc-core`

### Build log

```
CMake Error at CMakeLists.txt:150 (find_package):
  Could not find a configuration file for package "OpenCASCADE" that is
  compatible with requested version "7.6.2".

  The following configuration files were considered but not accepted:

    /nix/store/sfjfl0vbv207b024638wn8xgbnwrg0nz-opencascade-occt-7.8.1/lib/cmake/opencascade/OpenCASCADEConfig.cmake, version: 7.8.1
```

### Additional context

Add any other context about the problem here.

### Notify maintainers

@gebner 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -12494,6 +12494,7 @@ self: super: with self; {
     inherit (pkgs) fontconfig rapidjson;
     inherit (pkgs.xorg) libX11 libXi libXmu libXext;
     inherit (pkgs.darwin.apple_sdk.frameworks) Cocoa;
+    opencascade-occt = pkgs.opencascade-occt_7_6;
   });
 
   python-olm = callPackage ../development/python-modules/python-olm { };

